vue-frontend/src/components/mobile-app-development/Services.vue

```diff
@@ -0,0 +1,201 @@
+import Csharp from "@/assets/images/mobile-app-development/services/ios/Csharp.webp";
+import flutter from "@/assets/images/mobile-app-development/services/flutter/flutter.webp";
+import dart from "@/assets/images/mobile-app-development/services/flutter/dart.webp";
+
+export default {
+  data() {
+    return {
+      services: [
+        {
+          id: 1,
+          title: "Android App Development",
+          image: android,
+          technologies: [
+            {
+              image: gradle,
+              name: "GRADLE",
+            },
+            {
+              image: kotlin,
+              name: "KOTLIN",
+            },
+            {
+              image: java,
+              name: "JAVA",
+            },
+            {
+              image: clang,
+              name: "C++",
+            },
+          ],
+          description:
+            "Take your business to new heights with an Android app! Tap into a global user base, boost customer engagement, communicate directly, and gather insightful data. Plus, enhance your visibility, add revenue streams, and provide top-notch customer service. With over a decade of expertise in Android app development, our team can help you craft high-quality, user-friendly apps that drive business growth and customer engagement.",
+        },
+        {
+          id: 2,
+          title: "iOS App Development",
+          image: ios,
+          technologies: [
+            {
+              image: swift,
+              name: "SWIFT",
+            },
+            {
+              image: objectiveC,
+              name: "OBJECTIVE C",
+            },
+            {
+              image: javaScript,
+              name: "JAVASCRIPT",
+            },
+            {
+              image: Csharp,
+              name: "C#",
+            },
+          ],
+          description:
+            "Our seasoned iOS developers specialize in creating aesthetically pleasing, intuitive, and user-friendly apps that seamlessly function across all Apple devices. We leverage the latest iOS technologies to deliver solutions that give you an edge in the competitive Apple app market. Plus, our team follows Human Interface Guidelines defined by Apple to deliver exceptional user experience.",
+        },
+        {
+          id: 3,
+          title: "Flutter App Development",
+          image: flutter,
+          technologies: [
+            {
+              image: dart,
+              name: "DART",
+            },
+          ],
+          description:
+            "Bring your multi-platform app vision to life with our Flutter app development services. We excel in crafting high-quality, stunning applications that work flawlessly on Android, iOS, and the web, all from a single codebase, facilitating faster time to market and cost savings.",
+        },
+      ],
+    };
+  },
+};
+</script>
+<style scoped>
+.gradient-border {
+  border-image: linear-gradient(to bottom, #f2709c, #ff835b) 1;
```

Comment:
you can add this class to tailwind.config and use it.

Reply:
we can add background image like that not border image tailwind is not providing way to do it